sql/expression/function/spatial/st_dimension_test.go

```diff
@@ -151,7 +151,7 @@ func TestDimension(t *testing.T) {
 		require.NoError(err)
 
 		typ := f.Type()
-		_, err = typ.Convert(v)
+		_, _, err = typ.Convert(v)
```

Comment:
Minor: in tests, it may be a good idea to check the 2nd result.

Reply:
this might be directionally correct but i only refactored numbers, these are geometry types. they aren't capable of setting the out of range bool

Reply:
is someone implements geometry out of range they should would probably write tests for it at that point